### PR TITLE
Add Prometheus 2.2.1

### PR DESCRIPTION
**Pull request by gongled**
Hello, @andyone.

I ask you to pack Prometheus 2.2.1 and publish it into EK repository. I do not know how to deal with logging directory: it writes only to stdout/stderr and does not have any options to output logging to file without redirects using UNIX pipe. I have checked RPM package locally on CentOS 6 and CentOS 7 and it looks fine, but with no logging at all.

### CODE REVIEW

**Comment by andyone** [comment]
Hi! 

`kv.daemonize` it is wrapper for `daemonize` utility and it supports stdout/stderr redirection. For this, you should define [`kv[log]`](https://docs.kaos.st/kaosv/latest/#267) variable with the path to log file.

File: prometheus/prometheus.spec [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+URL:              https://prometheus.io
+
+Source0:          https://github.com/%{name}/%{name}/archive/v%{version}.tar.gz
+Source1:          %{name}.service
+Source2:          %{name}.init
+Source3:          %{name}.sysconfig
+
+BuildRoot:        %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+
+BuildRequires:    golang >= 1.9
```
Comment (andyone) [inline prometheus/prometheus.spec]: Define latest available Go version (1.10).

File: prometheus/prometheus.spec [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+install -pm 0644 documentation/examples/%{name}.yml %{buildroot}%{_sysconfdir}/%{name}/%{name}.yml
+install -pm 0644 %{SOURCE3} %{buildroot}%{_sysconfdir}/sysconfig/%{name}
+
+%if 0%{?rhel} >= 7
+    install -pm 0644 %{SOURCE1} %{buildroot}%{_unitdir}/%{name}.service
+%else
+    install -pm 0755 %{SOURCE2} %{buildroot}%{_initrddir}/%{name}
+%endif
+
+%check
```
Comment (andyone) [inline prometheus/prometheus.spec]: Empty `%check` section.

File: prometheus/prometheus.spec [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+%endif
+
+Conflicts:       prometheus < 2.0
+
+Provides:        %{name} = %{version}-%{release}
+
+################################################################################
+
+%description
+
```
Comment (andyone) [inline prometheus/prometheus.spec]: Redundant empty line.

**Review by andyone — CHANGES_REQUESTED** [review]

File: prometheus/prometheus.spec (hunk 2 above)
Comment (andyone) [reply prometheus/prometheus.spec]: I will add check for this to `perfecto`.

File: prometheus/prometheus.spec (hunk 1 above)
Comment (gongled) [reply prometheus/prometheus.spec]: Fixed (5fbb62c3d01b1ee56ba6ecadf54db7f3d5c157a1)

File: prometheus/prometheus.spec (hunk 3 above)
Comment (gongled) [reply prometheus/prometheus.spec]: Fixed (713772844f00c2c16151097e6c1cd102dddf3950)

File: prometheus/prometheus.spec (hunk 2 above)
Comment (gongled) [reply prometheus/prometheus.spec]: Fixed (17ae9ae54df2d89f4bee20225412d11859903292)

File: prometheus/SOURCES/prometheus.service [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+[Service]
+EnvironmentFile=-/etc/sysconfig/prometheus
+User=prometheus
+ExecStart=/usr/bin/prometheus \
+          --web.console.libraries=/usr/share/prometheus/console_libraries \
+          --web.console.templates=/usr/share/prometheus/consoles \
+          $PROMETHEUS_OPTS
+ExecReload=/bin/kill -HUP $MAINPID
+Restart=always
+
```
Comment (andyone) [inline prometheus/SOURCES/prometheus.service:19]: Add `StandardOutput` and `StandardError` for [output rediction](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#Logging%20and%20Standard%20Input/Output).

Comment (gongled) [reply prometheus/SOURCES/prometheus.service:19]: Oh, thank you. I missed it. Fixed (d2cecc47e49270226b0e0c01753ff49b825fbd98).

**Review by andyone — APPROVED** [review]

File: prometheus/prometheus.spec [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+%defattr(-,root,root,-)
+%{_bindir}/%{name}
+%{_bindir}/promtool
+%config(noreplace) %{_sysconfdir}/logrotate.d/%{name}
+%config(noreplace) %{_sysconfdir}/%{name}/%{name}.yml
+%{_datarootdir}/%{name}
+%if 0%{?rhel} >= 7
+    %{_unitdir}/%{name}.service
+%else
+    %{_initrddir}/%{name}
```
Comment (andyone) [inline prometheus/prometheus.spec]: Don't use prefix spaces in `%files` section.

**Review by andyone — CHANGES_REQUESTED** [review]

File: prometheus/prometheus.spec (hunk 5 above)
Comment (gongled) [reply prometheus/prometheus.spec]: Fixed (b3cacf056485e9b0e5baa5acf1caef634e2ac020).

**Review by andyone — APPROVED** [review]

**Comment by andyone** [comment]
Looks good, merged.

**Comment by andyone** [comment]
Packages were added to `kaos-testing` repository.

**Pull request merged**